tests/generation/test_generate_missing.py

```diff
@@ -38,11 +55,11 @@ def test_foo():
         @scenario("Code is generated for scenario steps which are not yet defined(implemented)")
         def test_missing_steps():
             pass
-    """
+        """
         )
     )
 
-    result = testdir.runpytest(dirname, "--generate-missing", "--feature", tests.join("generation.feature").strpath)
+    result = testdir.runpytest("--generate-missing", "--feature", "generation.feature")
 
```

Comment:
let's check that no test is run (result.assert_outcomes) and that status code is 0